packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -3287,29 +3287,27 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
   @mustCallSuper
   void didChangeDependencies() {
     assert(_active); // otherwise markNeedsBuild is a no-op
-    _debugCheckOwnerBuildTargetExists('didChangeDependencies');
+    assert(() {
+      _debugCheckOwnerBuildTargetExists('didChangeDependencies');
+      return true;
+    });
```

Comment:
if you do use a closure, we have to end by calling it now (`assert(() { ...; return true; }());`).

Reply:
Switched to no closure.